src/web/app/components/datepicker-today/datepicker-today.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { NgbCalendar } from '@ng-bootstrap/ng-bootstrap';
+import { DateFormat } from '../session-edit-form/session-edit-form-model';
+
+/**
+ * Datepicker with today button component
+ */
+@Component({
+  selector: 'tm-datepicker-today',
+  templateUrl: './datepicker-today.component.html',
+  styleUrls: ['./datepicker-today.component.scss'],
+})
+export class DatepickerTodayComponent implements OnInit {
```

Comment:
DatepickerToday sounds a bit off. Maybe just call it `DatePicker` instead?

Reply:
I want to avoid using `DatePicker` as it might be confused with the original one from bootstrap

Reply:
yes I agree